Rakefile

```diff
@@ -2,7 +2,7 @@ require 'rake/testtask'
 
 Rake::TestTask.new do |t|
   t.test_files = Dir.glob('spec/**/*_spec.rb')
-  t.libs = ["lib", "spec"]
+  t.libs = %w(lib spec)
```

Comment:
We actually prefer `%w[ ]` internally, can you configure rubocop to prefer that and use it instead?

```yaml
Style/PercentLiteralDelimiters:
  PreferredDelimiters:
    '%w': []
    '%W': []
```